### PR TITLE
Add default cluster role bindings

### PR DESCRIPTION
**Pull request by deads2k**
Add default cluster roles bindings to rbac bootstrapping.  Also adds a case for allowing escalation when you have no authenticator.

@liggitt I expect you may need to make peace with this.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/34019)

### CODE REVIEW

File: pkg/registry/rbac/escalation_check.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/auth/user"
+)
+
+func EscalationAllowed(ctx api.Context, superUser string) bool {
+	u, ok := api.UserFrom(ctx)
+	if !ok {
+		// the only way to be without a user is to either have no authenticators by explicitly saying that's your preference
```
Comment (deads2k) [inline pkg/registry/rbac/escalation_check.go]: @liggitt reasoning in this comment here.

**Comment by deads2k** [comment]
@k8s-bot gci gke e2e test this

**Comment by deads2k** [comment]
@k8s-bot gke e2e test this

File: pkg/registry/rbac/rest/storage_rbac.go [hunk 2]
```diff
@@ -131,5 +131,25 @@ func PostStartHook(hookContext genericapiserver.PostStartHookContext) error {
 		glog.Infof("Created clusterrole.%s/%s", rbac.GroupName, clusterRole.Name)
 	}
 
+	existingClusterRoleBindings, err := clientset.ClusterRoleBindings().List(api.ListOptions{})
+	if err != nil {
+		utilruntime.HandleError(fmt.Errorf("unable to initialize clusterrolebindings: %v", err))
+		return nil
+	}
+	// if clusterrolebindingss already exist, then assume we don't have work to do because we've already
```
Comment (liggitt) [inline pkg/registry/rbac/rest/storage_rbac.go]: clusterrolebindingss, my precioussss

**Comment by deads2k** [comment]
spelling fixed

**Pull request merged**